clients/src/main/java/org/apache/kafka/clients/consumer/internals/StreamsRebalanceData.java

```diff
@@ -244,23 +277,71 @@ public String toString() {
         }
     }
 
+    private final UUID processId;
+
+    private final Optional<HostInfo> endpoint;
+
+    private final Map<String, String> clientTags;
+
     private final Map<String, Subtopology> subtopologies;
 
     private final AtomicReference<Assignment> reconciledAssignment = new AtomicReference<>(Assignment.EMPTY);
 
-    public StreamsRebalanceData(Map<String, Subtopology> subtopologies) {
+    private final AtomicReference<Map<HostInfo, List<TopicPartition>>> partitionsByHost = new AtomicReference<>(Collections.emptyMap());
```

Comment:
This is fine for now, but when we migrate the IQv2 work into trunk it'll get updated this to hold active and standby `TopicPartition`.